beta/src/components/MDX/Challenges/Challenges.tsx

```diff
@@ -117,7 +117,9 @@ export function Challenges({children, isRecipes}: ChallengesProps) {
               'text-3xl mb-2 leading-10 relative',
               isRecipes ? 'text-purple-50 dark:text-purple-30' : 'text-link'
             )}>
-            {isRecipes ? 'Try out some recipes' : 'Try out some challenges'}
+            {isRecipes
+              ? '몇 가지 레시피를 시도해보세요.'
+              : '몇 가지 도전을 시도해보세요.'}
```

Comment:
```suggestion
              ? '몇 가지 레시피를 시도해 보세요.'
              : '몇 가지 도전을 시도해 보세요.'}
```
띄어쓰기 수정해 주시면 좋을 것 같습니다!